### PR TITLE
Update the link to input of build_gas_input_locations

### PR DESCRIPTION
**Pull request by yerbol-akhmetov**
## Changes proposed in this Pull Request
Good day. here I propose to update the link to the input of `build_gas_input_locations` rule as previous link is no longer available. FYI @martacki 

## Checklist

- [x] I tested my contribution locally and it works as intended.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] A release note `doc/release_notes.rst` is added.

### CODE REVIEW

**Comment by fneum** [comment]
Would it be ok for you @martacki, @yerbol-akhmetov to perhaps move `PyPSA:chp_cc` to a fork of yours as long as you are developing it? Then you can have a separate space for PRs against this branch.

File: envs/environment.yaml [hunk 1]
```diff
@@ -39,12 +39,12 @@ dependencies:
 - pyscipopt
 - matplotlib
 - proj
-- fiona
+- fiona=1.9.5
```
Comment (martacki) [inline envs/environment.yaml:42]: why do we need a fixed fiona version?

File: envs/environment.yaml [hunk 2]
```diff
@@ -39,12 +39,12 @@ dependencies:
...
 - country_converter
 - geopy
 - tqdm
 - pytz
-- tabula-py
+- tabula-py=2.7.0
```
Comment (martacki) [inline envs/environment.yaml:47]: why do we need a fixed tabula-py version?

**Comment by martacki** [comment]
> Would it be ok for you @martacki, @yerbol-akhmetov to perhaps move `PyPSA:chp_cc` to a fork of yours as long as you are developing it? Then you can have a separate space for PRs against this branch.

Hi @fneum
yes for future project we are already switching to our fork! This one here escalated a bit due to time constraints, initially we wanted to contribute only a small PR to upstream (hence the banch name). I will merge this one for now and we will rewind by opening smaller PRs to upstream. If @yerbol-akhmetov can save the existing branch on our fork, we can even delete it here.

Maybe some work can be "saved and contributed" to main afterwards

**Pull request merged**